addon/package.json

```diff
@@ -0,0 +1,158 @@
+  "dependencies": {
+    "@ember/test-waiters": "^3.0.2",
+    "@embroider/macros": "^1.10.0",
+    "@embroider/util": "^1.9.0",
+    "broccoli-debug": "^0.6.5",
+    "broccoli-funnel": "^3.0.8",
+    "ember-auto-import": "^2.6.0",
+    "ember-cli-babel": "^7.26.11",
+    "ember-cli-htmlbars": "^6.2.0"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.20.7",
+    "@babel/core": "^7.21.0",
+    "@babel/preset-typescript": "^7.21.0",
+    "@ember/optional-features": "^2.0.0",
+    "@ember/string": "^3.0.1",
+    "@glimmer/component": "^1.1.2",
+    "@glimmer/interfaces": "^0.84.1",
+    "@glimmer/reference": "^0.84.2",
+    "@glint/template": "^0.9.7",
+    "@tsconfig/ember": "^1.0.1",
+    "@types/jquery": "^3.5.16",
+    "@types/rsvp": "^4.0.4",
+    "@typescript-eslint/eslint-plugin": "^4.33.0",
+    "@typescript-eslint/parser": "^4.33.0",
+    "broccoli-babel-preset-typescript": "^1.0.1",
+    "broccoli-merge-trees": "^4.2.0",
+    "concurrently": "^7.6.0",
+    "documentation": "^14.0.1",
+    "ember-cli": "~4.9.2",
+    "ember-cli-dependency-checker": "^3.3.1",
+    "ember-cli-inject-live-reload": "^2.1.0",
+    "ember-cli-test-loader": "^3.0.0",
+    "ember-fetch": "^8.1.2",
+    "ember-in-element-polyfill": "^1.0.1",
+    "ember-load-initializers": "^2.1.2",
+    "ember-maybe-import-regenerator-for-testing": "^1.0.0",
+    "ember-resolver": "^9.0.1",
+    "ember-source": "~4.9.2",
+    "ember-source-channel-url": "^3.0.0",
+    "ember-try": "^2.0.0",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.6.0",
+    "eslint-plugin-disable-features": "^0.1.3",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "expect-type": "^0.15.0",
+    "fs-extra": "^11.1.0",
+    "latest-version": "^5.0.0",
+    "loader.js": "^4.7.0",
+    "pretender": "^3.4.7",
+    "prettier": "^2.8.1",
+    "qunit": "^2.19.4",
+    "release-it": "~15.6.0",
+    "release-it-lerna-changelog": "^5.0.0",
+    "rimraf": "^3.0.2",
+    "typescript": "^4.7.4",
+    "webpack": "^5.75.0"
+  },
+  "engines": {
+    "node": "16.* || >= 18"
+  },
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org/"
+  },
+  "changelog": {
+    "repo": "emberjs/ember-test-helpers",
+    "labels": {
+      "breaking": ":boom: Breaking Change",
+      "enhancement": ":rocket: Enhancement",
+      "bug": ":bug: Bug Fix",
+      "documentation": ":memo: Documentation",
+      "internal": ":house: Internal"
+    }
+  },
+  "ember": {
+    "edition": "octane"
+  },
+  "ember-addon": {
+    "configPath": "tests/dummy/config"
+  },
+  "release-it": {
+    "scripts": {
+      "after:bump": "yarn docs"
+    },
+    "plugins": {
+      "release-it-lerna-changelog": {
+        "infile": "CHANGELOG.md",
+        "launchEditor": true
+      }
+    },
+    "git": {
+      "tagName": "v${version}"
+    },
+    "github": {
+      "release": true,
+      "tokenRef": "GITHUB_AUTH"
+    }
+  },
+  "volta": {
+    "node": "16.19.0",
```

Comment:
Use `extends`? `node` version is now out of sync with the one defined in the root `package.json` file.

Reply:
can do